membership-common/src/main/scala/com/gu/services/model/PaymentDetails.scala

```diff
@@ -26,10 +27,11 @@ object PaymentDetails {
   case class Payment(price: Price, date: LocalDate)
   implicit def dateToDateTime(date: LocalDate): DateTime = date.toDateTimeAtStartOfDay()
 
-  def apply(
+  def fromSubAndPaymentData(
```

Comment:
renamed this, the confusion between the default apply method and this one could be easily resolved that way